index.js

```diff
@@ -189,7 +189,7 @@ WebpackBuildNotifierPlugin.prototype.onCompilationDone = function(results) {
         var error = results.compilation.errors[0];
         notify = true;
         title += 'Error';
-        msg = this.messageFormatter(error, error.module && error.module.rawRequest ? error.module.rawRequest : '');
+        msg = error && this.messageFormatter(error, error.module && error.module.rawRequest ? error.module.rawRequest : '');
```

Comment:
I presume we should do the same thing for the next code block to handle null/undefined `warning` as well?

Also would be nice to include a default error/warning message in the case when `!error` or `!warning`.

Reply:
Done. Please check.